pymagine/edge_detection.py

```diff
@@ -3,11 +3,14 @@
 @author: Trevor Kwan
 Implementation of edge_detection function in the pymagine package.
 """
+import numpy as np
+from skimage.color import rgb2gray
+from scipy.signal import convolve2d
+import matplotlib.pyplot as plt
 
```

Comment:
need cv2 package to write image
```suggestion
import cv2
```